custom_components/climate/xiaomi_miio.py

```diff
@@ -132,6 +132,7 @@ def __init__(self, hass, name, host, token, sensor_entity_id, customize):
         self._current_swing_mode = None
         self._current_operation = None
         self._current_fan_mode = None
+        self._is_on = False
```

Comment:
This isn't needed anymore if you use `self._state`. The initial state must be `None` to mark the device as unavailable. The first update call will initialize the state.